app/views/editor/_toolbar.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= javascript_tag("$(document).on('ContentLoad', function() { tfm.tools.initTypeAheadSelect($('#preview_host_id')) });"); %>
```

Comment:
I think this should move into the `if show_preview` block below, no point in running this if the field doesn't exist.